packages/terra-card/src/_themes.scss

```diff
@@ -0,0 +1,9 @@
+@import './mixins';
+
+@include cerner-consumer-theme {
+  @include terra-consumer-theme-card();
+  /* stylelint-disable selector-class-pattern */
+  .terra-consumer-theme-margin {
```

Comment:
I wouldn't solve this with a theme.... this should be solved with a parent layout component. Could be a grid, card-list (if we create one).